src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeaderSection/ContactHeaderAddressSection.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Link, styled, Typography } from '@material-ui/core';
+import { LocationOn } from '@material-ui/icons';
+import { Skeleton } from '@material-ui/lab';
+
+import React, { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+import theme from '../../../../../theme';
+import { ContactDetailsHeaderFragment } from '../ContactDetailsHeader.generated';
+import { ContactHeaderSection } from './ContactHeaderSection';
+
+interface Props {
+  loading: boolean;
+  contact?: ContactDetailsHeaderFragment;
```

Comment:
You could have a separate fragment for each of these sub components. That way each component specifies the fields it needs directly. Might be easier to maintain in the future.